src/shared/util.js

```diff
@@ -1433,13 +1433,19 @@ MessageHandler.prototype = {
     let targetName = data.sourceName;
     let capability = createPromiseCapability();
 
-    let sendStreamRequest = ({ stream, chunk, success, reason, }) => {
+    let sendStreamRequest = ({ stream, chunk, transfers,
+                               success, reason, }) => {
+      if (transfers) {
```

Comment:
I don't understand the concern, but sendStreamRequest will have optional transfers, and I recommend just call `this.postMessage({...}, transfers)` as mentioned above